digdag-standards/src/test/java/io/digdag/standards/operator/redshift/RedshiftUnloadOperatorFactoryTest.java

```diff
@@ -0,0 +1,118 @@
+package io.digdag.standards.operator.redshift;
+
+import com.amazonaws.auth.AWSSessionCredentials;
+import com.google.common.base.Optional;
+import com.google.common.collect.ImmutableMap;
+import io.digdag.spi.OperatorContext;
+import io.digdag.spi.TaskRequest;
+import io.digdag.standards.operator.jdbc.JdbcOpTestHelper;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class RedshiftUnloadOperatorFactoryTest
+{
+    private JdbcOpTestHelper testHelper = new JdbcOpTestHelper();
+    private RedshiftUnloadOperatorFactory operatorFactory;
+
+    @Rule
+    public ExpectedException thrown= ExpectedException.none();
+
+    @Before
+    public void setUp()
+    {
+        operatorFactory = testHelper.injector().getInstance(RedshiftUnloadOperatorFactory.class);
+    }
+
+    @Test
+    public void getType()
+    {
+        assertThat(operatorFactory.getType(), is("redshift_unload"));
```

Comment:
to make this line effective, I think that RedshiftLoadOperatorFactory and RedshiftUnloadOperatorFactory need to add secrets prefixed with `redshift_load` and `redshift_unload` to OperatorFactory.getSecretAccessList.